### ISSUE TITLE
Non Archive-It WASAPI Import

### ISSUE THREAD

**Issue by ianmilligan1** [archivesunleashed/auk#29]
Right now AUK works really well with Archive-It collections. It would be great to investigate if we could get collections into AUK from other sources, ideally using WASAPI. 

[WebRecorder.io](http://webrecorder.io/) would be ideal as we could then support individual researchers by ingesting their personally-created WARCs and provide analysis.

Right now, however, to get collection data we rely on the Internet Archive's Collections API.

**Comment by ruebot** [comment]
On Hold: Waiting on webrecorder.io to implement WASAPI.

**Comment by ruebot** [comment]
Doesn't look like webrecorder.io is going to do this implementation. Closing this issue for now.

**Comment by despens** [comment]
WASAPI export should work → https://webrecorder.io/docs/api
